src/llm/BUILD

```diff
@@ -87,7 +87,8 @@ cc_library(
         "//src:libovmsprofiler",
         "llmcalculator_cc_proto",
         ":llm_engine",
-    ],
+        "//src/python:utils",
+    ] + PYBIND_DEPS,

```

Comment:
we select clause for no-python build probably at place where llmnoderesources are mentioned as dependency.